src/components/widgets/AccountBalanceWidget.vue

```diff
@@ -89,16 +93,8 @@ export default {
 		},
 
 		mosaicName() {
-			const mosaicAlias = this.data.mosaic?.mosaicAliasNames[0];
-
-			if (mosaicAlias) {
-				const mosaicNamespaces = mosaicAlias.split('.');
-				const mosaicLastSubnamespace = mosaicNamespaces.pop();
-
-				return mosaicLastSubnamespace;
-			}
-
-			return this.data.mosaic?.mosaicId || http.networkCurrency.namespaceName.split('.').pop();
+			// Only display for network currency
+			return http.networkCurrency.namespaceName.split('.').pop();
 		},
```

Comment:
is this widget intended to only ever displaying currency now?

if so, why the check in balance()?

Reply:
I'm not sure I understand your question correctly.

> is this widget intended to only ever displaying currency now?
> if so, why the check in balance()?

yes, It was displayed the first mosaic from an array, XYM will always sort to the first one.
If the account holding XYM, that is not a problem, because it showed XYM and balance.

If the account does not hold any XYM, the other mosaic will be displayed.

To check balance is ensure only display XYM rather than other mosaics.

Reply:
i mean is AccountBalanceWidget intended to only ever show XYM balance or could it be used to show balance of other mosaics as well?

Reply:
yes only show XYM balance.